### ISSUE TITLE
ThorVG internal file error

### ISSUE THREAD

**Issue by Gangwa-Labs** [kisvegabor/lvgl#7214]
### LVGL version

v9.1

### What happened?

When using the internal ThorVG library to play a Lottie animation it fails to build due to errors inside the thorVG library

"no instance of constructor "ColorStop::ColorStop" matches the argument list" & "no match for 'operator=' (operand types are 'ColorStop' and '<brace-enclosed initializer list>')"

I've placed the line of code that is causing it to fail. it can be found on line 721 in lvgl/src/libs/thorvg/tvgLottieProperty.h

### How to reproduce?

            const_cast<LottieColorStop&>(other).value = {nullptr, nullptr};

**Comment by liamHowatt** [comment]
I could not reproduce the compilation error in `release/v9.1` or `master`. Are you able to update the LVGL? Otherwise, please send a more precise version description of LVGL which you are using. Thanks!

**Comment by C47D** [comment]
I tried to search a similar error in the thorVG repo but couldn't find one. 

---

HUGE OFFTOPIC:

From a "C head" this looks like the `ColorStop` struct doesn't have the `=`operator OR it doesn't match the expected `{ }` initialization.

**Comment by kisvegabor** [comment]
cc @hermet

**Comment by mesopa** [comment]
Also run into the same problem, tried with LVGL builds 9.2 and 9.2.2 and got always the same errors when enabling the ThorVG on my project, inside PlatformIO.

**Comment by hermet** [comment]
Hello, I guess this fix addresses the issue. Could someone verify it with this change?

721 in lvgl/src/libs/thorvg/tvgLottieProperty.h
```
//const_cast<LottieColorStop&>(other).value = {nullptr, nullptr};
const_cast<LottieColorStop&>(other).value = ColorStop();   
```

**Comment by DblBee** [comment]
I tested this fix and can confirm this solves the compile issue with LVGL.

`const_cast<LottieColorStop&>(other).value = ColorStop(); `

**Comment by hermet** [comment]
here is the patch: https://github.com/lvgl/lvgl/issues/7214, the fix will be in thorvg v0.15.5. thanks.

**Comment by kisvegabor** [comment]
Thank you @hermet!

@Gangwa-Labs can you send a PR to LVGL with the fix?
